docs/import.md

```diff
@@ -144,6 +144,16 @@ ollama push <your username>/example
 
 After publishing, your model will be available at `https://ollama.com/<your username>/example`.
 
+### Tagging models
+
+If your model has different versions you can optionally tag them, without a tag it will be `latest`:
```

Comment:
latest is a mutable tag, and not good practice.  We should advocate tagging explicitly, not using `latest` if possible.

Modern container repositories does not support mutable tag because of security.

Reply:
I agree it would probably be better to remove the optionally part. Given `latest` is mutable it will be overwritten if you don't specify a tag. I will see if I can't come up with some better wording to get that across. 👍